Stripe/PublicHeaders/UI/STPPaymentCardTextField.h

```diff
@@ -108,6 +108,11 @@
 @property(nonatomic, strong, nullable) UIView *inputAccessoryView;
 
 /**
+ *  The curent brand image displayed in the receiver.
+ */
+@property (nonatomic, readonly, nullable) UIImage *brandImage;
```

Comment:
This is exposed so that subclassers can use the image for their custom `brandImageRectForBounds:` logic.